#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_gcc4.3.dpatch by Martin Michlmayr
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Makes dansguardian compatible with GCC 4.3 (#41739)

@DPATCH@
diff -urNad dansguardian-2.8.0.6-antivirus-6.4.4.1~/FOptionContainer.cpp dansguardian-2.8.0.6-antivirus-6.4.4.1/FOptionContainer.cpp
--- dansguardian-2.8.0.6-antivirus-6.4.4.1~/FOptionContainer.cpp	2006-10-28 21:37:00.000000000 +0200
+++ dansguardian-2.8.0.6-antivirus-6.4.4.1/FOptionContainer.cpp	2007-04-02 09:53:54.000000000 +0200
@@ -22,6 +22,7 @@
 #include "FOptionContainer.hpp"
 #include "OptionContainer.hpp"
 #include "RegExp.hpp"
+#include <cstdlib>
 #include <string>
 #include <iostream>
 #include <fstream>
diff -urNad dansguardian-2.8.0.6-antivirus-6.4.4.1~/VirusScanner.hpp dansguardian-2.8.0.6-antivirus-6.4.4.1/VirusScanner.hpp
--- dansguardian-2.8.0.6-antivirus-6.4.4.1~/VirusScanner.hpp	2006-03-13 00:30:46.000000000 +0100
+++ dansguardian-2.8.0.6-antivirus-6.4.4.1/VirusScanner.hpp	2007-04-02 09:53:04.000000000 +0200
@@ -19,6 +19,7 @@
 // and some additions by mad@madness.at
 
 #include "autoconf/platform.h"
+#include <cstdlib>
 #include <stdexcept>
 #include <fstream>
 #include <cerrno>
